src/components/Details/VmDetails/VmDetails.js

```diff
@@ -0,0 +1,252 @@
+import React, { Fragment } from 'react';
+import PropTypes from 'prop-types';
+import { forEach, get, has } from 'lodash';
+import { FieldLevelHelp } from 'patternfly-react';
+import { PodModel, VirtualMachineInstanceMigrationModel, VirtualMachineModel } from '../../../models';
+import { VmStatus } from '../../VmStatus';
+
+const DASHES = '---';
+const VIRT_LAUNCHER_POD_PREFIX = 'virt-launcher-';
+const IMPORTER_DV_POD_PREFIX = 'importer-datavolume-';
+
+const cloudInitInUse = vm => {
```

Comment:
you can use https://github.com/kubevirt/web-ui-components/blob/master/src/utils/selectors.js#L26 which was merged today

Reply:
Done!